canonical-web-and-design/ubuntu.com
# Remove kubcon19 notice

## Done

Remove kubcon19 notice from homepage as the even is over

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the notice is no longer there

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2023
